### PR TITLE
Added driver order confirmation. Added specs

### PR DESCRIPTION
**Pull request by domenic99**
#38

### CODE REVIEW

File: app/controllers/orders_controller.rb [hunk 1]
```diff
@@ -17,11 +17,17 @@ def create
...
   end
 
+  def show
+    render json: { 'order' => Order.find(params[:id]) }
+  end
+
   def update
+    return render json: { 'error' => 'You are not a driver' }, status: 422 unless @current_user.instance_of? Driver
     order = Order.find(params[:id])
-    order.driver_id = Driver.find(2).id # should be current driver
```
Comment (rakvium) [inline app/controllers/orders_controller.rb]: :sweat_smile:

Comment (domenic99) [inline app/controllers/orders_controller.rb]: hmm, wrong commit :D

Comment (domenic99) [inline app/controllers/orders_controller.rb]: That was imitation of acception. I needed it when I was doing email sending.

File: app/controllers/orders_controller.rb [hunk 2]
```diff
@@ -17,17 +17,27 @@ def create
...
+    order.driver_id = @current_user.id
     order.state = 'in progress'
     order.save
-    client_email = Client.find(order.client_id).email
-    ClientMailer.welcome_email(params[:id], client_email).deliver_now if client_email
+    render json: { 'current order' => order.id }
```
Comment (rakvium) [inline app/controllers/orders_controller.rb]: @domenic99 I think [it's better not to use spaces in JSON keys, as far as it complicates the process of parsing on the front end side by prohibiting usage of the dot notation](http://stackoverflow.com/questions/10311361/accessing-json-object-keys-having-spaces).
Also I suggest you to respond with the full JSON of the order.

**Pull request merged**